content/en/observability_pipelines/set_up_pipelines/archive_logs/datadog_agent.md

```diff
@@ -233,7 +233,7 @@ Follow the instructions for the cloud provider you are using to archive your log
 
 ### Install the Observability Pipelines Worker
 1. Select your platform in the **Choose your installation platform** dropdown menu.
-1. Enter the Datadog Agent address. This is the address and port where your Datadog Agent is sending its logging data. The Observability Pipelines Worker listens to this address for incoming logs.
+1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port, for example `0.0.0.0:<port_number>`, for incoming logs from the Datadog Agent.
```

Comment:
```suggestion
1. Enter the Datadog Agent address. The Observability Pipelines Worker listens to this address and port. For example, `0.0.0.0:<port_number>` for incoming logs from the Datadog Agent.
```

Reply:
Thanks @rtrieu! I'd like to keep second part with the first part of the sentence, what do you think of this?

> The Observability Pipelines Worker listens to this address and port for incoming logs from the Datadog Agent. For example, `0.0.0.0:<port_number>`.

Reply:
I think that works even better!